### ISSUE TITLE
Print function: Possibility of integrating a print.css

### ISSUE THREAD

**Issue by klauswagner** [humhub/meeting-issues#5]
Customize the print output by including a individual print.css (in html-head of /meeting/index/print?id=xy), either on a fixed path or by specifying the path in the module settings.

**Comment by olekristensen** [comment]
Yes, please, i change the css in my browser development view every time i export a pdf for distribution of minutes to the members...

**Comment by buddh4** [comment]
What kind of changes do you add in your print style? Maybe we can just improve the included print style?

**Comment by klauswagner** [comment]
> Customize the print output by including a individual print.css (in html-head of /meeting/index/print?id=xy), either on a fixed path or by specifying the path in the module settings.

We are currently adding a line to `/protected/modules/meeting/views/index/print.php`:
`<link rel="stylesheet" type="text/css" href="/themes/xyz/css/meeting_print.css" />`

In our meeting_print.css we are currently defining things such as spacing, colors, additional style elements, etc. with 130 lines of code so that the print output fits the corporate design.

So too much and too individually to generally implement this in the module. Therefore, the option of specifying a print.css in the module settings or through a defined file path within the active theme would be useful.
